web/controllers/post_controller.ex

```diff
@@ -38,7 +38,8 @@ defmodule Tilex.PostController do
            |> Repo.preload([:channel])
            |> Repo.preload([:developer])
 
-    render(conn, "show.html", post: post)
+    Plug.Conn.assign(conn, :canonical_url, "https://til.hashrocket.com" <> Tilex.Router.Helpers.post_path(conn, :show, post))
```

Comment:
Like I mentioned on Slack. I didn't pay enough attention and notice this was being used in app layout. Sorry 😞 .  So I believe we can use `post_url(conn, :show, post)` here instead of `"https://til.hashrocket.com" <> Tilex.Router.Helpers.post_path(conn, :show, post)`.